examples/models/llama/source_transformation/quantize.py

```diff
@@ -72,12 +72,37 @@ def quantize(  # noqa C901
     if qmode == "int8":
         # Add quantization mode options here: group size, bit width, etc.
         return WeightOnlyInt8QuantHandler(model).quantized_model()
-    elif qmode.startswith("torchao:"):
+    elif qmode.startswith("torchao:fpa"):
+        pattern = r"torchao:fpa(\d+)w"
+        matches = re.findall(pattern, qmode)
+        assert len(matches) == 1, f"Expected 1 match for pattern but got {len(matches)}"
+        bitwidth = int(matches[0][0])
+        _load_torchao_aten_lib(
+            libname="libtorchao_ops_mps_linear_fp_act_xbit_weight_aten"
```

Comment:
I thought you had a nice lib name libtorchao_ops_mps? Why this long name?

Reply:
done!